openapi/index.yaml

```diff
@@ -1529,3 +1561,75 @@ definitions:
     required:
       - tag
       - rptId
+  MessageStatus:
+    type: object
+    properties:
+      status:
+        $ref: '#/definitions/MessageStatusValue'
+      updated_at:
+        $ref: '#/definitions/Timestamp'
+      version:
+        type: integer
+    required:
+      - status
+      - updated_at
+  MessageStatusAttributes:
```

Comment:
Do we have to maintain this?

Reply:
Yes, it's used above in EnrichedMessage.